components/org.wso2.carbon.identity.sso.saml.common/src/main/java/org/wso2/carbon/identity/sso/saml/common/Util.java

```diff
@@ -151,8 +157,9 @@ public static String getUserNameFromOpenID(String openid) throws IdentityExcepti
      *
      * @param userName User name
      * @return OpenID corresponding the given user name.
-     * @throws org.wso2.carbon.identity.base.IdentityException
+     * @throws org.wso2.carbon.identity.base.IdentityException this is deprecated because this repo is saml and this is a openid method
```

Comment:
this is a wrong place to add a deprecation comment!

Reply:
fixed